src/components/BookingForm/BookingForm.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { createNewVespa, fetchVespas } from '../../redux/slices/vespaSlice';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './BookingForm.css';
+
+const BookingForm = () => {
```

Comment:
- [ ] When I go to the "booking ride" on the mobile view, and then open the navbar with the hamburger menu, this happens:

1. I go to "Book a Vespa"

![image](https://github.com/Rudy-Menekam/Rentals-project-frontend/assets/57421823/05104cd6-f15c-4669-8328-1bde0a215a5e)

2. I open the menu.  The menu appears to be overflowing out of the screen.

![image](https://github.com/Rudy-Menekam/Rentals-project-frontend/assets/57421823/8d177ace-53f6-4a01-93a2-9e05bf31dca2)

Kindly, fix this styling so the menu appears correctly when on the "Book a Vespa" section. 👍